client/landing/jetpack-cloud/components/backup-activity-log/activity-type-selector.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React, { ChangeEvent, FunctionComponent, useState, useRef } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ActivityTypeCount } from './types';
+import { Button } from '@automattic/components';
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormLabel from 'components/forms/form-label';
+import Popover from 'components/popover';
+
+interface Props {
+	activityTypeCounts: ActivityTypeCount[];
+	hiddenActivities: string[];
+	setHiddenActivities: ( hiddenActivityKeys: string[] ) => void;
+}
+
+const BackupsActivityTypeSelector: FunctionComponent< Props > = ( {
+	activityTypeCounts,
+	hiddenActivities,
+	setHiddenActivities,
+} ) => {
+	const translate = useTranslate();
+
+	const [ showSelector, setShowSelector ] = useState( false );
+	const buttonRef = useRef( null );
+
+	const togglePopover = () => setShowSelector( ! showSelector );
+
+	const onChange = ( { target: { name, checked } }: ChangeEvent< HTMLInputElement > ) => {
+		setHiddenActivities(
+			checked ? hiddenActivities.filter( key => key !== name ) : hiddenActivities.concat( name )
+		);
+	};
+
+	return (
+		<>
+			<Button ref={ buttonRef } onClick={ togglePopover }>
+				{ translate( 'Activity type' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Activity Type' )` *ES Score: 10*